ods_ci/tests/Tests/500__jupyterhub/elyra-pipelines.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Documentation    Test Suite for Elyra pipelines in the SDS image
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/Elyra.resource
+Resource         ../../Resources/Page/ODH/Pipelines/Pipelines.resource
+Resource         ../../Resources/Page/OCPDashboard/Builds/Builds.robot
+Library          Screenshot
+Library          DebugLibrary
+Library          JupyterLibrary
+Suite Teardown   End Web Test
+Force Tags       JupyterHub
+
+
+*** Variables ***
+${SVG_CANVAS} =    //*[name()="svg" and @class="svg-area"]
+${SVG_INTERACTABLE} =    /*[name()="g" and @class="d3-canvas-group"]
+${SVG_PIPELINE_NODES} =    /*[name()="g" and @class="d3-nodes-links-group"]
+${SVG_SINGLE_NODE} =    /*[name()="g" and contains(@class, "d3-draggable")]
+
+
+
+*** Test Cases ***
+Verify Pipeline Is Displayed Correctly
```

Comment:
As pipelines can be also displayed in dashboard, could you rename it to `Verify Pipeline Is Displayed Correctly In Elyra` ?